vue-components/src/components/Popover.vue

```diff
@@ -159,7 +180,7 @@ $base: '.wikit-Popover';
 		box-sizing: border-box;
 	}
 
-	&--bottom {
+	&--main-wrapper {
```

Comment:
I'm not sure what the idea behind this is.
Shouldn't all those values be different for at least one the directions?
If there are some declaration that are the same for all, then why not add them to their respective classes above?